backend/video/views.py

```diff
@@ -182,16 +182,29 @@ def list_recent(request):
     # Get the query param from the request, default count is 10
     count = int(request.query_params.get("count", 10))
 
-    # Get a list of videos from recently transcribed videos
+    # Note: Currently, we have implemented this get recent method based on the logic that
+    # one Transcript of either type ORIGINAL_SOURCE or type MACHINE_GENERATED
+    # will always have one video associated with it.
+    # In the future, if that constraint is removed then we might need to alter the logic.
+
+    # Get the 2n latest transcripts from the DB for the user
```

Comment:
It's `1n` for now. In the future, if the above constraint is removed, we may have to make it `2n`

Reply:
Yes, I will update that.